src/js/apps/globals/app-frame_app.js

```diff
@@ -13,6 +13,36 @@ import ProgramsMainApp from 'js/apps/programs/programs-main_app';
 
 import { AppNavView, AppNavCollectionView } from 'js/views/globals/app-nav/app-nav_views';
 
+const topNavMenu = new Backbone.Collection([
+  {
+    onSelect() {
+      Radio.trigger('event-router', 'patients:all');
+    },
+    id: 'your-workspace',
+    isFas: false,
+    icon: 'window',
+    text: 'globals.appNav.topMenu.workspace',
+  },
+  {
+    onSelect() {
+      Radio.trigger('event-router', 'programs:all');
+    },
+    id: 'program-admin',
+    isFas: true,
+    icon: 'tools',
+    text: 'globals.appNav.topMenu.admin',
+  },
+  {
+    onSelect() {
+      Radio.request('auth', 'logout');
+    },
+    id: 'sign-out',
+    isFas: true,
```

Comment:
couldn't get the `itemTemplateContext` in the OptionList to work for whatever reason, so did this instead

Reply:
works for now :+1: